### ISSUE TITLE
Multiclass Error (string labels)

### ISSUE THREAD

**Issue by MLombriz** [vecxoz/vecstack#38]
I have the following error: "could not convert string to float: 'H'". In my problem I have 3 classes 'H','D','A'. I can't find if its possible to do the stacking model. 
Can you help me with this issue?
This is the code is was running:

RANDOM_model = RandomForestClassifier(class_weight=None,criterion='entropy',max_depth=17, max_features='auto',
                                      max_leaf_nodes=32,min_samples_leaf= 1,min_samples_split= 2,n_estimators=54,random_state= 1)
LR_model = linear_model.LogisticRegression(C= 10000000000.0,class_weight= None,fit_intercept= True,multi_class='auto',
                                             penalty='l2',solver='liblinear')

models = [RANDOM_model,LR_model]
Strain, Stest = stacking(models,Xtrain, ytrain, Xtest,  regression=False, mode='oof_pred_bag', 
                         needs_proba=False,save_dir=None, metric=accuracy_score,n_folds=5, 
                         random_state=1,stratified=False,shuffle=True,verbose=2)

model_stack = XGBClassifier(colsample_bytree= 1,gamma= 0.5,learning_rate= 1,max_depth= 2,min_child_weight=1,n_estimators= 11,random_state= 1)

model_stack = model_stack.fit(Strain, ytrain)
y_pred_stack = model_stack.predict(Stest)
print('Final prediction score: [%.8f]' % accuracy_score(ytest, y_pred_stack))

**Comment by vecxoz** [comment]
Hi,
Thanks for using vecstack.

The reason for this error is that vecstack expects numerical class labels (not strings).
So you need to convert your class labels using `sklearn.preprocessing.LabelEncoder`.
When you get predictions you can transform them back to strings using method `inverse_transform`.
More details: [https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html).
